src/apps/AppCenter/InstalledApps/AppCardContent.js

```diff
@@ -2,89 +2,99 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import color from 'onecolor'
 import { Badge, Button, font, theme } from '@aragon/ui'
-import { AppCenterAppType } from '../../../prop-types'
+import defaultAppIcon from '../../../assets/default-app-icon.svg'
+import RemoteImage from '../../../components/RemoteImage'
+import { RepoType } from '../../../prop-types'
+import { appIconUrl } from '../../../utils'
 
-const AppCardContent = ({
-  app: { appName, name, icons, description, canUpgrade },
-  onOpen,
-}) => (
-  <section
-    css={`
-      display: flex;
-      flex-direction: column;
-      height: 100%;
-      padding-top: 24px;
-      justify-content: space-between;
-    `}
-  >
-    <div
+const AppCardContent = ({ repo, onOpen }) => {
+  const { name, repoName, baseUrl, currentVersion, latestVersion } = repo
+  const { description, icons } = latestVersion.content
+  const canUpgrade = currentVersion.version !== latestVersion.version
+  const iconUrl = appIconUrl({ baseUrl, icons })
+  return (
+    <section
       css={`
-        flex-grow: 2;
-        margin-bottom: 20px;
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        padding-top: 24px;
+        justify-content: space-between;
       `}
     >
-      <img
-        alt=""
-        src={icons.large}
-        width="56"
-        height="56"
+      <div
         css={`
-          display: block;
-          margin: 0 auto 20px;
-          width: 56px;
-          height: 56px;
-        `}
-      />
-      <h1
-        css={`
-          text-align: center;
-          margin-bottom: 8px;
-          ${font({ size: 'large', weight: 'bold' })};
+          flex-grow: 2;
+          margin-bottom: 20px;
         `}
       >
-        {name}
-      </h1>
-      {canUpgrade && (
-        <div
+        <RemoteImage src={iconUrl}>
```

Comment:
We should use `AppIcon` here, so we get the radius applied too (I can look into it).